[IMP] accounting/l10n_de: update to celebrate GoBD-compliance (#11417)
odoo/documentation: content/applications/finance/fiscal_localizations/germany.rst

hosting, paperless companies, etc.).

.. important::
   Odoo is certified **GoBD-compliant**.

What do you need to know about GoBD when relying on accounting software?
------------------------------------------------------------------------

.. note::
   If you can, the best way to understand GoBD is to read the `official GoBD text
   <https://ao.bundesfinanzministerium.de/ao/2021/Anhaenge/BMF-Schreiben-und-gleichlautende-Laendererlasse/Anhang-64/anhang-64.html>`_.

>    It is a bit long but quite readable for non-experts. But in short, here is what to expect:

Reviewer: I would remove this note entirely since I added a seealso at the beginning of the section and I find the second sentence a bit weird for documentation.

Author: I removed the note completely. But I am not sure what you are refering to here:

> I added a seealso at the beginning of the section

Should I add sth else?

   (`l10n_de_pos_cert`) and **Germany - Certification for Point of Sale of type restaurant**
   (`l10n_de_pos_res_cert`) modules.

   .. tip::
      If these modules are not listed, :ref:`update the app list <general/install>`.

.. image:: germany/pos-upgrade.png
   :align: center
   :alt: Upgrading Odoo Point of Sale from the Apps dashboard

Register your company at the financial authority
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

To register your company, go to :menuselection:`Settings --> General Settings --> Companies -->
Update Info`, fill out the following fields and *Save*.

- **Company name**
- Valid **address**
- **VAT** number
- **St.-Nr** (Steuernummer): this number is assigned by the tax office to every taxable natural or
  legal person. (e.g., `2893081508152`)
- **W-IdNr** (Wirtschafts-Identifikationsnummer): this number is used as a permanent
  identification number for economically active persons.

You can then **register your company through fiskaly** by opening the *fiskaly* tab and clicking on
the *fiskaly Registration* button.

.. image:: germany/fiskaly-registration.png
   :align: center

>    :alt: Button to register a company through fiskaly in Odoo

Reviewer: I would remove this image as it's not really needed

.. tip::
   If you do not see the *fiskaly Registration* button, make sure that you *saved* your company
   details and are not in *editing mode* anymore.

> Once the registration has been finalized, new fields appear:
> 
> - **fiskaly organization ID** refers to the ID of your company at the fiskaly side.
> - **fiskaly API key** and **secret** are the credentials the system uses to access the services
>   offered by fiskaly.
> 
> .. image:: germany/fiskaly-keys.png
>    :align: center
>    :alt: fiskaly keys as displayed on Odoo

Reviewer: ```suggestion
Once the registration has been finalized, the following fields appear:

- :guilabel:`Fiskaly organization ID`: refers to the ID of your company on fiskaly;
- :guilabel:`Fiskaly API Key` and :guilabel:`Fiskaly API Key`: credentials used by the system to access the services offered by fiskaly.
```
I would also remove this screenshot

.. note::
   It is possible to request new credentials if there is any issue with the current ones.

Create and link a Technical Security System to your PoS
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. image:: germany/create-tss.png
   :align: right

>    :alt: Create TSS option from a point of sale

Reviewer: I would remove this screenshot

To use your point of sale in Germany, you first have to create a :abbr:`TSS (Technical Security
System)` for it.

> To do so, go to :menuselection:`Point of Sale --> Configuration --> Point of Sale`, open the point
> of sale you want to edit, then check the box next to **Create TSS** and *Save*.

Reviewer: ```suggestion
To do so, :ref:`access the POS settings <configuration/settings>`, select your POS`, scroll down to the :guilabel:`Fiskaly API section`, enable :guilabel:`Create TSS`, and :guilabel:`Save`.
```

.. image:: germany/tss-ids.png
   :align: right

>    :alt: Example of TSS ID and Client ID from fiskaly in Odoo Point of Sale

Reviewer: Same for this one

Once the creation of the TSS is successful, you can find your **TSS ID** and **Client ID** under the
*fiskaly API* section.

- **TSS ID** refers to the ID of your TSS at fiskaly's side.

> - **Client ID** refers to your PoS but at fiskaly's side.

Reviewer: Once the TSS has been created, fill in the fields:

- :guilabel:`TSS ID`: refers to the ID of your TSS on fiskaly.
- :guilabel:`Client ID`: refers to your PoS on fiskaly.

Author: Done; but I did `Once the TSS has been created, the following fields appear:` for the first sentence instead.
The fields are only shown after the creation process and are pre-filled automatically with the just created values.
What do you think?

DSFinV-K
--------

.. image:: germany/dsfinv-k-export.png
   :align: right

>    :alt: Menu to export DSFinV-K

Reviewer: I would also remove this image

Whenever you close a PoS session, the orders' details are sent to the :abbr:`DSFinV-K (Digitale
Schnittstelle der Finanzverwaltung für Kassensysteme)` service of fiskaly.

> In case of an audit, you can export the data sent to DSFinV-K by going to :menuselection:`Point of
> Sale --> Orders --> DSFinV-k exports`.
> 
> These fields are mandatory:
> 
> - **Name**
> - **Start Datetime** (export data with dates larger than or equal to the given start date)
> - **End Datetime** (export data with dates smaller than or equal to the given end date)
> 
> Leave the **Point of Sale** field blank if you want to export the data of all your points of sale.
> Specify a Point of Sale if you want to export this specific PoS' data only.

Reviewer: ```suggestion
In case of an audit, you can export the data sent to DSFinV-K by following these steps:

 #. Go to :menuselection:`Point of Sale --> Orders --> DSFinV-k exports` and click :guilabel:`New`.
 #. Fill in the :guilabel:`Start Datetime` and :guilabel:`End Datetime` fields.
 #. Select a :guilabel:`Point of Sale` or leave the field empty to export the data of all your points of sale.
```

> The creation of a DSFinV-K export triggers on export at fiskaly's side.

Reviewer: ```suggestion
The creation of a DSFinV-K export triggers an export on fiskaly's side.
```

.. image:: germany/dsfinv-k-export-fields.png
   :align: center

>    :alt: Pending DSFinV-K export on Odoo

Reviewer: Can you please update this screenshot?

Author: done; I cropped out the button though

> As you can see, the **State** is *Pending*. This means that the export has been successfully
> triggered and is being processed. You have to click on *Refresh State* to check if it is ready.

Reviewer: ```suggestion
The :guilabel:`State` is then set to :guilabel:`Pending`. This means that the export has been successfully
triggered and is being processed. Click :guilabel:`Refresh State` to check if it is ready.
```
